<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Choose OS (Exclusive)</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="winCb">
          <property name="text">
           <string>Window</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="macCb">
          <property name="text">
           <string>Mac</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="linuxCb">
          <property name="text">
           <string>Linux</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Choose HDD (Non-Exclusive)</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="oneTbCb">
          <property name="text">
           <string>1 TB</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="twoTbCb">
          <property name="text">
           <string>2 TB</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="threeTbCb">
          <property name="text">
           <string>3 TB</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Choose SSD</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="samsungRb">
        <property name="text">
         <string>Samsung</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="logictechRb">
        <property name="text">
         <string>Logitech</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="miRb">
        <property name="text">
         <string>Mi</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="grabDataBtn">
       <property name="text">
        <string>Grab Data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="setDefaultStateBtn">
       <property name="text">
        <string>Set Default State</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
